server/src/main/java/org/elasticsearch/cluster/metadata/Metadata.java

```diff
@@ -1818,7 +1818,7 @@ public Builder() {
             this(Map.of(), 0);
         }
 
-        Builder(Metadata metadata) {
+        public Builder(Metadata metadata) {
```

Comment:
I think we want this to remain protected, favoring `.builder(otherMeta)` instead

Reply:
Oh good point! I had missed that that method existed already.